tests/scripts/all.sh

```diff
@@ -2982,11 +2982,13 @@ component_test_make_shared () {
 
 component_test_cmake_shared () {
     msg "build/test: cmake shared" # ~ 2min
+    root_dir="$(pwd)"
     cmake -DUSE_SHARED_MBEDTLS_LIBRARY=On .
     make
     ldd programs/util/strerror | grep libmbedcrypto
     make test
     programs/test/dlopen_demo.sh
+    PKG_CONFIG_PATH="${root_dir}/pkgconfig" ${root_dir}/tests/scripts/pkgconfig.sh
```

Comment:
Hmm, this is failing CI on FreeBSD, so shouldn't be done there I guess

Reply:
Yeah it should be skipped there, I'll have to figure out how to do that

Reply:
Ok I have it skipping non-linux systems now for this port, lets see if the CI is happy